frontend/packages/dev-console/src/components/import/route/CreateRoute.tsx

```diff
@@ -1,26 +1,17 @@
 import * as React from 'react';
-import * as _ from 'lodash';
 import { useFormikContext, FormikValues } from 'formik';
 import { TextInputTypes } from '@patternfly/react-core';
-import { InputField, DropdownField } from '@console/shared';
-import { makePortName } from '../../../utils/imagestream-utils';
+import { InputField } from '@console/shared';
+import RouteInputField from './RouteInputField';
 
 const CreateRoute: React.FC = () => {
   const {
     values: {
       image: { ports },
-      route: { defaultUnknownPort, targetPort },
+      route: { defaultUnknownPort },
     },
   } = useFormikContext<FormikValues>();
-  const portOptions = ports.reduce((acc, port) => {
-    const name = makePortName(port);
-    acc[name] = (
-      <>
-        {port.containerPort} &rarr; {port.containerPort} ({port.protocol})
-      </>
-    );
-    return acc;
-  }, {});
+  const portOptions = ports.map((port) => port.containerPort);
```

Comment:
When you convert the ports here to strings the `Select` automatically matches the user input and do not show "Create "8080"" if the option is already available. Wdyt?

```suggestion
  const portOptions: string[] = ports.map((port) => port.containerPort.toString());
```

![image](https://user-images.githubusercontent.com/139310/94488428-e28c1500-01e2-11eb-884a-650a2d9ae74f.png)

Reply:
Updated. Not sure if there is something we can do about the "Create ..." option, even the Patternfly demos behave the same way: https://www.patternfly.org/v4/documentation/react/components/select#typeahead

Reply:
Okay, thanks for revalidating this. I though that the .toString() solves this issue.